strawberry/codegen/types.py

```diff
@@ -55,6 +63,8 @@ class GraphQLFragmentType(GraphQLObjectType):
     fields: List[GraphQLField] = field(default_factory=list)
     graphql_typename: Optional[str] = None
     on: str = ""
+    definition_node: Optional[Node] = None
+    strawberry_type: Optional[StrawberryType] = None
```

Comment:
I wonder about the naming here :) I think I'd make something like this:

```suggestion
    graphql_node: Optional[Node] = None
    graphql_type: Optional[StrawberryType] = None
```

what do you think?

Reply:
Sure, I'm not picky about the naming here.

Reply:
let's do the change then 😊